src/Terminus.php

```diff
@@ -114,6 +120,7 @@ public function run(InputInterface $input, OutputInterface $output)
      */
     private function addBuiltInCommandsAndHooks()
     {
+/*
         $commands = $this->getCommands([
```

Comment:
We can't use `getCommands()` et. al. in the phar because filesystem iteration functions don't work in the phar filesystem. The command discovery class is still used for plugins, though.